### DISCUSSION TITLE
挂在的百度网盘，无法加载及下载

### DISCUSSION THREAD

**Discussion by xjsyy** [AlistGo/alist#4610] [General]
Failed to fetch http://127.0.0.1:5244/p/test/jihuo%20.txt?sign=oww_ADlyerlKLA9Iop_EAkkPGmCCpzFe6kejbR1-emY=:0: AxiosError: Network Error
![屏幕截图 2023-06-18 090436](https://github.com/alist-org/alist/assets/118407022/6f39e1bf-41a7-4127-988a-58215f40472f)
[挂在配置：下载接口用的非官方，开启了Web代理,WebDav策略用的本地代理，签名已启用。]
下载接口如果用官方，可以下载和加载，但是非常慢。（应该大家都这样吧）

**Comment by xukeguo** [comment]
代理地址如何写

**Reply by xjsyy** [reply]
![image](https://github.com/alist-org/alist/assets/118407022/e44dd046-ac1c-4c83-b730-ff332440624b)
我只是这样弄了一下。。。。
